.github/workflows/release.yml

```diff
@@ -147,3 +147,11 @@ jobs:
           repository: observIQ/otel-collector-installer-testing
           event-type: upstream_prerelease
           client-payload: '{ "version": "${{ steps.get-tag.outputs.tag }}" }'
+      # Trigger amplify repo
+      - name: Update Amplify Version
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.ORG_GORELEASER_GITHUB_TOKEN }}
+          repository: observIQ/amplify
+          event-type: update_collector_version
```

Comment:
i think it probably makes the most sense to just call this event the same as the other (upstream_prerelease)